esbuild.js

```diff
@@ -22,6 +22,7 @@ const options = {
   target: ['node18'],
   minify: !dev,
   sourcemap: dev,
+  loader: { '.node': 'file' },
```

Comment:
Would you please check that the native modules are _not_ copied over into the build output for functions that do not use gcn-kafka --- that is to say, that they are tree-shaken?

Reply:
It was getting copied into the build folder directly, I updated it in the latest commit and it no longer gets copied. Also just tested and successfully produced and consumed